src/lib/src/ng-select.component.ts

```diff
@@ -94,6 +101,10 @@ export class NgSelectComponent implements OnInit, OnDestroy, ControlValueAccesso
                 private renderer: Renderer2
     ) {
         this.mergeConfig(config);
+        this._itemsSubscription = Observable.combineLatest(this._items$, this._value$).subscribe((res) => {
+            this.setItems(res[0]);
```

Comment:
There are still issues with this approach
1) every time you change selected model, items get repopulated even if it didn't change
2) if you re-set items, component selects old writeValue

Reply:
1. Yes, but this will happen very only few times, in most cases only one time when component is loaded. If I change model using dropdown this logic is not called.

2. This will also happen in most cases only once and write value still need to be recalculated.

Reply:
not once, every time user changes items or selected model programatically